examples/ws2812/main.go

```diff
@@ -0,0 +1,38 @@
+// Connects to an WS2812 RGB LED strip with 10 LEDS, such as
+// on an Adafruit Circuit Playground Express board.
+//
+// Replace machine.NEOPIXELS in the code below to match the pin
+// that you are using, if you have a different board.
+package main
+
+import (
+	"image/color"
+	"machine"
+	"time"
+
+	"github.com/tinygo-org/drivers/ws2812"
+)
+
+func main() {
+	neo := machine.GPIO{machine.NEOPIXELS}
+	neo.Configure(machine.GPIOConfig{Mode: machine.GPIO_OUTPUT})
+
+	ws := ws2812.New(neo)
+	leds := make([]color.RGBA, 10)
+	rg := false
+
+	for {
+		rg = !rg
+		for i := range leds {
+			rg = !rg
+			if rg {
+				leds[i] = color.RGBA{R: 0xff, G: 0x00, B: 0x00, A: 0x77}
+			} else {
+				leds[i] = color.RGBA{R: 0x00, G: 0xff, B: 0x00, A: 0x77}
```

Comment:
`A` is ignored on WS2812 so perhaps it's best to leave it out in the example, otherwise people will get confused.